.env

```diff
@@ -9,3 +9,4 @@ DJANGO_SETTINGS_MODULE=eahub.config.settings
 LEAN_MANAGERS="Local Manager=leanmanager@example.com"
 GROUPS_EMAIL=groups@example.com
 ADMINS="Local Admin=admin@example.com"
+FEEDBACK_URL="https://feedback.eahub.org"
```

Comment:
@sebgrebe, do you think it should be an env variable?

Reply:
I thought that's what you suggested on the previous MR on this? (I might misremember this though). 
If not, what would you suggest instead?

Reply:
maybe you're talking about URL reversing? sadly there's no way to reverse that one

In general I don't recommend to store in envs anything except secrets and environment variables, it has downsides eg as cluttering and inaccessibility. With django when something must be configurable it's better to move it out into db using a specific package as https://github.com/lazybird/django-solo

Reply:
Added django solo and removed env var